staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -850,7 +846,7 @@ func parse(selector string) (internalSelector, error) {
 		return nil, err
 	}
 	sort.Sort(ByKey(items)) // sort to grant determistic parsing
-	return internalSelector(items), err
```

Comment:
why is it unnecessary?

Reply:
> why is it unnecessary?

Variable `items` is  internalSelector type, so no need to trans.